client/src/login/loginReducer.js

```diff
@@ -0,0 +1,16 @@
+import { Map } from 'immutable';
+
+export const SAVE_USER = 'SAVE_USER';
```

Comment:
I think the important thing is that we do this consistently but I do have an argument for why we should put these consts in the reducer file rather than the actions file: reducers are the ones processing these symbols and that's kind of all they do. Every single type is going to be in the reducer's switch statement. Actions are a bit broader and it seems like they don't even have to produce an event if they don't want to. It feels like the reducer should be the owner of them.

Anyway, not super important, we can do it either way but I would prefer it this way.